### ISSUE TITLE
`Messages.get` does not work when delivered/opened/clicked are empty.

### ISSUE THREAD

**Issue by maramizo** [trycourier/courier-python#79]
```
3 validation errors for ParsingModel[MessageDetails]
__root__ -> delivered
  field required (type=value_error.missing)
__root__ -> opened
  field required (type=value_error.missing)
__root__ -> clicked
  field required (type=value_error.missing)
```

@

```

    def get(self, message_id: str, *, request_options: typing.Optional[RequestOptions] = None) -> MessageDetails:
        # ....
        if 200 <= _response.status_code < 300:
            return pydantic_v1.parse_obj_as(MessageDetails, _response.json())  # type: ignore
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ problematic line
```

**Comment by dsinghvi** [comment]
@maramizo thanks for filing this issue, we will re-generate the SDK shortly!

**Comment by dsinghvi** [comment]
@maramizo we recently published `v6.1.0` of the SDK which will remove hard failures like the one you just saw. Apologies for the inconvenience!

**Comment by maramizo** [comment]
Interesting approach to remove the explicit type safety by Pydantic though. Does that not defeat the purpose of having Pydantic types in the first place? If these are `None`, this will just offload errors which expect values into my codebase.
